### PR TITLE
Split files which test for the property order of functions

### PR DESCRIPTION
**Pull request by anba**
So SpiderMonkey can run the other parts of these tests.

The SpiderMonkey bug for this issue is <https://bugzilla.mozilla.org/show_bug.cgi?id=1629803>.

### CODE REVIEW

File: test/language/statements/class/definition/fn-length-static-precedence.js [hunk 1]
```diff
@@ -79,4 +75,3 @@ class E {
 }
 
 assert.sameValue(typeof E.length, 'function');
-assert(compareArray(Object.getOwnPropertyNames(E), ['length', 'name', 'prototype']))
```
Comment (rwaldron) [inline test/language/statements/class/definition/fn-length-static-precedence.js:82]: Need to remove `includes: [compareArray.js]`

File: test/language/statements/class/definition/fn-name-static-precedence.js [hunk 2]
```diff
@@ -76,4 +72,3 @@ class E {
 }
 
 assert.sameValue(typeof E.name, 'function');
-assert(compareArray(Object.getOwnPropertyNames(E), ['length', 'name', 'prototype']))
```
Comment (rwaldron) [inline test/language/statements/class/definition/fn-name-static-precedence.js:79]: Need to remove `includes: [compareArray.js]`

**Review by rwaldron — CHANGES_REQUESTED** [review]

File: test/language/statements/class/definition/fn-length-static-precedence.js (hunk 1 above)
Comment (anba) [reply test/language/statements/class/definition/fn-length-static-precedence.js:82]: Removed both includes.

**Review by rwaldron — APPROVED** [review]

**Comment by rwaldron** [comment]
@anba thanks!

**Pull request merged**